hacks/retail-analytics/qwiklabs.yaml

```diff
@@ -1,8 +1,8 @@
 schema_version: 2
 default_locale: en
-title: "[gHacks] Retail Analytics"
+title: "[gHacks] Real-time Analytics with CDC"
 description: "This hack will take you through replicating and processing operational data from 
-  an Oracle database into Google Cloud in real time"
+  an Oracle database into Google Cloud in real time using Change Data Capture (CDC)"
```

Comment:
Here you expanded the acronym, that's good. It just needs to be on the front blurb too.